wiki/Main_Page/en.md

```diff
@@ -3,9 +3,12 @@ layout: "main_page"
 ---
 
 <div class="wiki-main-page__blurb">
-Welcome to osu!, a free rhythm game created by peppy with its primary focus of clicking circles along with the beat. Aside from the main game mode, which is referred to many as *osu!standard*, osu! has three other different game modes, namely: osu!taiko, osu!catch, and osu!mania.
-<br/>
+<p>
+Welcome to osu!, a free-to-win rhythm game with its primary focus of clicking circles along with the beat. Aside from the main game mode, which is referred to many as <em>osu!standard</em>, osu! has three other different game modes, namely: osu!taiko, osu!catch, and osu!mania.
```

Comment:
you can render markdown within html by putting 3 newlines before it, don't use \<p\> and \<em\>

Reply:
The <p> was to also prevent a class from applying a bold font weight... will that be ok?

Reply:
not sure what u mean by bold, the font weight of __blurb is 300

and if u dont want that style then why is this whole thing in that element :thinking:

Reply:
easy way to resolve this is rewrite the paragraph... idk why it's talking about game modes right off the bat, and the second paragraph is leftover from the exact thing we wanted to replace

Reply:
Just to reiterate here, if we have the markdown parser parse the paragraph, it will put the paragraph content inside a \<p> with the `wiki-main-page__paragraph` class inside the blurb \<div>. That class will override the blurb's styles, setting the font-weight to 700 and font-style to italics. Any paragraph we put in this page will do the same because the `layout: "main_page"` assumes that all \<p> must have the `wiki-main-page__paragraph` class.

This seems more like an issue that osu-web would need to fix. That is, the `wiki-main-page__paragraph` should not be applying bold and italics (wiki will do it instead).